### PR TITLE
Partial fix to scrolling issues. No longer jumps up from the bottom when RHS opens

### PR DESCRIPTION
**Pull request by crspeller**

### CODE REVIEW

File: web/react/components/post_list.jsx [hunk 1]
```diff
@@ -105,18 +105,18 @@ export default class PostList extends React.Component {
         UserStore.addStatusesChangeListener(this.onTimeChange);
         SocketStore.addChangeListener(this.onSocketChange);
 
-        var postHolder = $(React.findDOMNode(this.refs.postlist));
+        const postHolder = $(React.findDOMNode(this.refs.postlist));
 
-        $(window).on('resize.' + this.props.channelId, function resize() {
+        $(window).resize(() => {
```
Comment (jwilander) [inline web/react/components/post_list.jsx]: Isn't there a corresponding `.off()` that should be modified too?

Comment (crspeller) [inline web/react/components/post_list.jsx]: Yes there was. Fixed.

**Comment by jwilander** [comment]
+1

**Comment by coreyhulen** [comment]
+1 waiting for build to pass

**Pull request merged**